server/errors.json

```diff
@@ -1398,5 +1398,15 @@
     "help": "",
     "url": "",
     "deprecates": ""
+  },
```

Comment:
Do we need a new one? Just use normal invalid stream name I think here.

Reply:
That's what I had done initially but then the error message the user gets is just "invalid stream name" but you don't know which stream name (the stream's name, the mirror name, one of the sources' name), so I added those two errors (invalid mirror and invalid source name). You prefer just a single one?

Reply:
ok you raise a good point from user experience point of view.